src/lesson-2/functions/getTaskById.ts

```diff
@@ -0,0 +1,7 @@
+import { data } from '../parsedData';
+import { Task } from '../types';
+
+export const getTaskById = (taskId: string): Task | null => {
+  const find = data.find((item) => item.id === taskId);
+  return find || null;
```

Comment:
окремо повертати `null` якщо елемент не знайдено це зайве, `.find` і так повертає undefined, цього достатньо
```suggestion
  return find;
```